packages/components/src/date-time/index.js

```diff
@@ -9,7 +9,8 @@ import 'react-dates/initialize';
  * WordPress dependencies
  */
 import { useState, forwardRef } from '@wordpress/element';
-import { __, _x } from '@wordpress/i18n';
+// import { __, _x } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
```

Comment:
It means that it will expose a new WP module under `wp-react-18n` handle that expose `wp.reactI18n` global.

Reply:
Is that something we don't want to do for some reason?

Reply:
I would like to understand first what benefits it brings to use the React hook rather than importing the same methods directly from the package. If we were to start using `useI18n` we should use it consistently everywhere.

Reply:
I'd like to understand that too. My only notion about this is that it's more "react-y" but that's not a good justification.

@jsnajdr @sirreal care to weigh in here?

Reply:
>  I'll just rescope it to just include the fix to the jest configuration so that typescript modules are detected by the tests. That's really the important finding in this PR anyway.

That would be fantastic 💯